zone/raids.cpp

```diff
@@ -1741,6 +1778,19 @@ void Raid::VerifyRaid()
 			else {
 				m.member = nullptr;
 			}
+			if (m.mainmarker == 1)
```

Comment:
Please enclose brackets on all of these checks please

Reply:
Updated